README.md

```
@@ -0,0 +1,140 @@
+**Programming that reads like plain English.**
+
+WFL is a natural-language programming language: you write what you want to do in
+words, and it runs. It aims to be a genuine *first* language for newcomers while
+staying powerful enough for production — with no cliff in between.
+
+```wfl
+store name as "World"
+display "Hello, " with name with "!"
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)